### ISSUE TITLE
Use tab to choose items fills the next item instead of the selected one

### ISSUE THREAD

**Issue by Merlin2001** [sblask/webextension-simple-form-fill#58]
### New Issue Checklist (tick off like this: `[x]`)

- [x] I checked existing issues (open and closed) for possible duplicates
- [x] I can reproduce the problem on the latest stable version (not nightly!)

### What is your browser?

- [x] Firefox
- [ ] Chrome (verified on Chrome, too)

### What is your operating system?

- [ ] Linux
- [ ] Mac
- [x] Windows (verfied on Mac, too)

### Description (please include examples/screenshots where applicable)

[Edit for clarification]  
With the following settings active:  
![2020-12-07_11h15_34](https://user-images.githubusercontent.com/13134791/101338735-aaa5eb80-387d-11eb-9323-887a4695ae36.png)
[/Edit]

When using tab to select an item from the fill suggestions list, the item with index + 1 is pasted instead. (If you select the last item in the list, nothing is filled into the form.)

See this screencast (<Tab> is pressed after selecting an item):  
![Use tab to choose items fills next item](https://user-images.githubusercontent.com/13134791/101167550-55c66300-363a-11eb-90fd-0211996c05c7.gif)

I have confirmed the behavior in Firefox (83.0) on macOS (10.13) and Windows (10-2004) and in Chrome (87.0.4280.88) on Windows.

**Comment by sblask** [comment]
Has this worked before? Because I didn't touch this in a long time. It works for me on Chrome 87, so I suspect a browser problem...

**Comment by Merlin2001** [comment]
You are right, Sebastian, I clarified the setting that causes the issue in my original post.

Judging from its tooltip (which I only discovered after posting, sorry 🙈) it is _somewhat_ intended behavior ("Tab will behave like Down, Shift-Tab like Up key, but only when the autocomplete popup is open"), although it doesn't seem to be possible to actually move through the list this way as (Shift)-Tab just always selects the (previous) next item.

I'd vote for removing the "Use tab to choose items" setting entirely, since it might be confusing for new users (like me - I thought that I _need_ to enable it, to be able to select autocomplete list items via <Tab> (which is not the case)) and doesn't work as advertised.
